setup.cfg

```diff
@@ -13,13 +13,13 @@ license_files =
   LICENSE.txt
 
 [options]
-python_requires = >= 3.7
+python_requires = >= 3.8
 setup_requires =
   setuptools >= 40.6
   pip >= 10
 include_package_data = true
 packages = find:
 install_requires =
-    Django>=3.2, <4.2
-    wagtail>=4.1, <5.0
+    Django>=3.2
+    wagtail>=5.2
```

Comment:
**Suggestion:** I think it would still be a good idea to restrict the upper versions here so that we're not claiming compatibility for future, possibly-breaking versions.

Reply:
Hi @alxbridge ,
Thanks, I set the upperbound to 6.1: https://github.com/torchbox/wagtail-makeup/pull/24/commits/720b2e6e0b6c1696dabacc128e13059743e43146

Reply:
I disagree. This is what nightly or weekly tests against Wagtail main should be for - pre-emptively finding failures.

In lots of cases, things will just work, so then you don't need to worry about making a new release. With this, we're increasing the maintenance burden

Reply:
@zerolab ,
Thanks for looking. I reverted it here:
https://github.com/torchbox/wagtail-makeup/pull/24/commits/a427c5e5ae3f19499d8d2b68926bd824eb40c355